src/dotnet/CarbonAware.DataSources.WattTime/RegionConverter.cs

```diff
@@ -0,0 +1,15 @@
+﻿using CarbonAware.Tools.WattTimeClient.Model;
+
+namespace CarbonAware.DataSources.WattTime;
+
+/// <summary>
+/// Class RegionConverter.
+/// </summary>
+public class RegionConverter : IRegionConverter
+{
+    /// <inheritdoc />
+    public Task<BalancingAuthority?> ConvertAzureRegionAsync(string region)
+    {
+        throw new NotImplementedException();
```

Comment:
Wait, but aren't we using this call later on? Is there a commit missing from this?

Reply:
There's a story to implement this.  This is just a stub for now.